datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/__init__.py

```diff
@@ -4,12 +4,13 @@
 import click
 
 from ...console import CONTEXT_SETTINGS
+from .build_traps_db import build_traps_db
 from .generate_profile import generate_profile_from_mibs
 from .translate_profile import translate_profile
 from .validate_mib_filenames import validate_mib_filenames
 from .validate_snmp_profiles import validate_profile
 
-ALL_COMMANDS = [generate_profile_from_mibs, translate_profile, validate_mib_filenames, validate_profile]
+ALL_COMMANDS = [generate_profile_from_mibs, build_traps_db, translate_profile, validate_mib_filenames, validate_profile]
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Should we use the same word for profile and traps_db ? In other words, should they both be `generate...` or `build` ?

Reply:
Renaming to generate, good idea